barman/backup_executor.py

```diff
@@ -1825,21 +1825,19 @@ def check(self, check_strategy):
         :param CheckStrategy check_strategy: the strategy for the management
              of the results of the various checks
         """
-        check_strategy.init_check("pgespresso extension")
+        check_strategy.init_check("postgres minimal version")
         try:
             # We execute this check only if the postgres connection is non None
             # and the server version is lower than 9.6. On latest PostgreSQL
             # there is a native API for concurrent backups.
-            if self.postgres and self.postgres.server_version < 90600:
-                if self.postgres.has_pgespresso:
-                    check_strategy.result(self.server_name, True)
-                else:
-                    check_strategy.result(
-                        self.server_name,
-                        False,
-                        hint="required for concurrent "
-                        "backups on PostgreSQL %s" % self.postgres.server_major_version,
-                    )
+            if self.postgres and not self.postgres.is_minimal_postgres_version():
+                check_strategy.result(
+                    self.server_name,
+                    False,
+                    hint="unsupported PostgresSQL version %s"
```

Comment:
It might be helpful to the user to also include the minimum supported version in this hint.

Reply:
Included